.github/workflows/circleci.yml

```
@@ -10,7 +10,8 @@ jobs:
     name: Run CircleCI artifacts redirector
     steps:
       - name: GitHub Action step
-        uses: | #v1.0.0
+        uses:
```

We want to remove all whitespace:
```suggestion
        uses: >-
```

---

According to yaml converters, looks like what is there _should_ work, not sure we need to change it, though `>-` would have also been viable to strip the whitespace.